app.py

```diff
@@ -412,7 +410,7 @@ def parse():
         'adventure_name': body.get('adventure_name', None)
     })
     if flag == 1:
```

Comment:
What does "flag == 1" mean? Does it mean the content is enabled?

Reply:
It means that we will unlock the next adventure when the flag == 1